ethminer/main.cpp

```diff
@@ -474,10 +474,10 @@ class MinerCLI
             << "    URL takes the form: scheme://user[.workername][:password]@hostname:port[/...]."
             << endl
             << "    where can be any of : " << endl
-            << "    http     for getWork mode" << endl
-            << "    tcp      for stratum mode" << endl
-            << "    tcps     for secure stratum mode" << endl
-            << "    tcpss    for secure stratum mode with strong TLS12 verification" << endl
+            << "    getwork     for getWork mode" << endl
+            << "    stratum     for stratum mode" << endl
+            << "    stratums    for secure stratum mode" << endl
+            << "    stratumss   for secure stratum mode with strong TLS12 verification" << endl
```

Comment:
i would change this to:

``for secure stratum mode with allowing only TLS 1.2 encryption``

Reply:
Isn't it the same statement ?

Reply:
Well yes and no.. it says TLS12 so there is a version 12 now? :) .. 
TLS12 is the internal name for TLS 1.2 only within boost. 
(https://www.boost.org/doc/libs/1_65_1/boost/asio/ssl/context_base.hpp)

Reply:
``ssl``  itself as in secure socket layer is no longer secure and has been replaced by TLS 1.0, 1.1 and 1.2

So supporting the real ssl modes does not make sense in my opinon.

Reply:
If a pool implements old and weak tls 1.0 we might end up accepting it.

Reply:
> If a pool implements old and weak tls 1.0 we might end up accepting it.

yea, i noticed that too. i'm also not sure what the default of boost is. so it can even support older ssl stuff.  

Maybe it makes sense to disable ssl and tls 1.0 (no_sslv2 , no_sslv3 + no_tlsv1)

Also in the past (initially) there were 3 modes.  TLS1.2, boost-default, allow-self-signed
Some of the error messages even hint to the allow-self-signed variant, which is no longer possible.

Reply:
Right! But even tls 1.0 is more secure that plain tcp.

Reply:
Should we also upgrade [scripts/testpools.bash](https://github.com/ethereum-mining/ethminer/blob/master/scripts/testpools.bash) ?

Reply:
@StefanOberhumer Actually no as it may be used to test previous versions also.